pkg/instrumentation/helper.go

```diff
@@ -43,3 +50,116 @@ func isAutoInstrumentationInjected(pod corev1.Pod) bool {
 	}
 	return false
 }
+
+// Check if single instrumentation is configured for Pod and return which is configured.
+func isSingleInstrumentationEnabled(insts languageInstrumentations) (bool, string) {
```

Comment:
For the new functions that take an `languageInstrumentations` can we add the functions to the struct?  This will prevent the need for reflection.